### PR TITLE
Update accept.txt

### PR DESCRIPTION
**Pull request by Braundo**
adding 'allowlist' and 'allowlisted' to acceptable terms

Signed-off-by: aaron <aaronbraundmeier@gmail.com>

#### :heavy_check_mark: Checklist

- [ ] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [ ] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

**Comment by freben** [comment]
I only see one of the words added though! You wanted both right?

Are you adding these separately to lay the ground work for the other PR that actually introduces the changes? You can add the terms directly in that same PR instead. That way you don't have to rebase that PR on master after waiting for a merge here.

**Comment by Braundo** [comment]
`allowlisted` is probably the only non-english word that I envision we'd use as replacements - everything else should be a "real" word :)

**Comment by freben** [comment]
Alright, let's merge then :)

**Review by freben — APPROVED** [review]

**Pull request merged**
